### PR TITLE
Feat/algolia integration

### PR DESCRIPTION
**Pull request by alexa-w**

### CODE REVIEW

**Comment by alexa-w** [comment]
tbh it's ugly and MVP oriented, but :woman_shrugging:

**Comment by alexa-w** [comment]
most searches won't work rn because the join table plant_tag was storing NULLs for id_plant

**Comment by alexa-w** [comment]
I've manually changed some entries in plant_tag in the DB to have id_plant for easier testing

File: server/index.js [hunk 1]
```diff
@@ -147,13 +151,12 @@ app.get('/user/login', (req, res) => {
 });
 
 app.get('/plant/category', (req, res) => {
-  dbHelpers.getPlantsByTags(req.query.category, (err, imageUrl) => {
-    console.log(req.query.category);
+  dbHelpers.getPlantsByIntersectionZipTag(req.query.zipcode, req.query.tag, (err, plants) => {
     if (err) {
       console.log(err);
       res.status(500).send('COULD NOT RETRIEVE IMAGE');
```
Comment (cskladz) [inline server/index.js:157]: Do you want to edit this console?

**Comment by cskladz** [comment]
Awesome!

**Pull request merged**